test/verify/check-system-info

```diff
@@ -380,5 +380,34 @@ class TestSystemInfo(MachineCase):
         b.enter_page("/system")
 
 
+    @skipImage("cockpit-system doesn't have the install-on-demand feature", "rhel-7-5")
+    def testPcpAvailable(self):
+        m = self.machine
+        b = self.browser
+
+        self.login_and_go("/system")
+
+        # images which have cockpit-pcp (most of them) should not show the "enable pcp" link
+        if not m.atomic_image and m.image not in ["debian-stable"]:
+            b.wait_visible("#server-pmlogger-onoff-row")
+            b.wait_not_visible("#system-information-enable-pcp")
+
+            # remove the cockpit-pcp package
+            m.execute("pkcon remove -y pcp")
+
+        if m.image in ["centos-7", "rhel-7"] or m.image.startswith("debian") or m.image.startswith("ubuntu"):
+            b.reload()
+            b.enter_page("/system")
+            b.wait_present("#server-pmlogger-onoff-row")
+
+        # ensure that the "enable pcp" link is visible
+        b.wait_not_visible("#server-pmlogger-onoff-row")
+        b.wait_visible("#system-information-enable-pcp")
+        b.wait_in_text("#system-information-enable-pcp", "Enable persistent metrics…")
+        b.click("#system-information-enable-pcp-link")
+        b.wait_present("#cockpit_modal_dialog")
+        b.wait_in_text("#cockpit_modal_dialog .modal-header", "Install Software")
```

Comment:
Oh right, this would fail because the VMs are offline. You could use test/verify/packagelib.py, derive your test from `PackageCase`, and `createPackage()` a fake `cockpit-pcp` to test it all the way through. See `class TestStoragePackages` as an example, I'm happy to walk you through it.